news/2013-09-06-august-2013-highlights.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "August 2013 Highlights"
+permalink: http://osunews.tumblr.com/post/60438764327/august-2013-highlights
+date: 2014-09-06 00:00:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/60438764327/august-2013-highlights
+---
+August was a hectic month for osu!, with the largest yet DDoS attack launched against us. We have fixed things up and are now on top of it all, though!
+
+### A new version-based changelog display is out!
+
+It's now much easier to see what features and bugfixes were in each release! After an update, you can click the notification in the lower right corner of the game client, or go to the main changelog and click one of the buttons up top with the number of the version you want to check out. This will bring up a list of every fix pushed out to that public release, which is quite a step up from the normal changelog when you want to know what updates have just gone live!  
+  
+
```

Comment:
Kind of a nitpick, but why does some articles have two consecutive newlines on some places?

Reply:
That's how <http://osunews.tumblr.com/post/60438764327/august-2013-highlights> had it. Probably for spacing in either the old website or tumblr.